README.md

```diff
@@ -207,12 +210,20 @@ API | Description | Auth | HTTPS | Link |
 | RandomUser | Generates random user data | No | Yes | [Go!](https://randomuser.me) |
 | RandomWord | Generate random word | No | No | [Go!](http://www.setgetgo.com/randomword/) |
 | ReqRes | A hosted REST-API ready to respond to your AJAX requests | No | Yes | [Go!](https://reqres.in/ ) |
-| Restpack | Provides screenshot, HTML to PDF, and content extraction APIs | `apiKey` | Yes | [Go!](https://restpack.io/) |
 | RoboHash | Generate random robot/alien avatars | No | Yes | [Go!](https://robohash.org/) |
+| Scrape Website Email | Grabs email addresses from a URL | `X-Mashape-Key` | Yes | [Go!](https://market.mashape.com/tommytcchan/scrape-website-email) |
+| SHOUTCLOUD | ALL-CAPS AS A SERVICE | No | No | [Go!](http://shoutcloud.io/) |
 | StackExchange | Q&A forum for developers | `OAuth` | Yes | [Go!](https://api.stackexchange.com/) |
 | UI Names | Generate random fake names | No | Yes | [Go!](https://github.com/thm/uinames) |
 | Verse | Check what's the latest version of your favorite open-source project | No | Yes | [Go!](https://verse.pawelad.xyz/) |
 
+### Dictionaries
+API | Description | Auth | HTTPS | Link |
+|---|---|---|---|---|
+| Oxford | Dictionary Data | `apiKey` | Yes | [Go!](https://developer.oxforddictionaries.com/) |
+| Wordnik | Dictionary Data | No | No | [Go!](http://developer.wordnik.com) |
```

Comment:
Confirmed Wordnik is still working and free. Got my API key today.

Reply:
Is there documentation?

Reply:
The docs link at the top of that page: http://developer.wordnik.com/docs.html

Should I change the link for this API?

Reply:
Should have been more specific -  is there some page that says that they are free?

Reply:
Googled around and found this page: http://developer.wordnik.com/#!/faq

Which says:

```
I'm pretty sure my app or site is going to be a HUGE hit and need a lot more than your free 15k calls/hour.
```

So it looks like it is documented as free 😄